### ISSUE TITLE
doc: docstrings

### ISSUE THREAD

**Issue by laszewsk** [cloudmesh/cloudmesh-cc#5]
provide docstrings to all methods in cloudmesh-cc, whennver you work on something start doing them.

**Comment by laszewsk** [comment]
has this been completed including service.py

**Comment by jpfleischer** [comment]
yes. https://groups.google.com/g/cloudmesh/c/z2nT4fnD9ro/m/IRzif4bxAwAJ

i believe this should be closed but i await your review.
